src/node/desktop/forge.config.js

```diff
@@ -46,6 +46,7 @@ const config = {
   packagerConfig: {
     icon: './resources/icons/RStudio',
     appBundleId: 'com.rstudio.desktop',
+    extendInfo: '../../cpp/desktop/Info.plist.in',
```

Comment:
We can't directly use this file, it needs to be consumed after cmake has generated the output file (Info.plist) via [configure_file](https://cmake.org/cmake/help/latest/command/configure_file.html#command:configure_file). Currently this doesn't happen for the Electron build (it's in the Qt-desktop branch of the cmakefile): https://github.com/rstudio/rstudio/blob/88d5f9b1359d261f56a8721ca1bc03d13fbc9701/src/cpp/desktop/CMakeLists.txt#L423-L424

There are some substitutions that happen in this file, so we do need to let cmake process it before we consume it. One example: https://github.com/rstudio/rstudio/blob/88d5f9b1359d261f56a8721ca1bc03d13fbc9701/src/cpp/desktop/Info.plist.in#L270

Reply:
We could either create another copy of this file that lives in the electron project, and have the electron cmake file process this, or do it from the qt desktop location. Eventually we'll be removing the Qt folder altogether, but so we'll need to resolve such overlaps, but this wouldn't be the first one.

Reply:
Note that we already do a bit of Electron stuff from the Qt desktop folder, see the end of the CMakelists.txt file: https://github.com/rstudio/rstudio/blob/88d5f9b1359d261f56a8721ca1bc03d13fbc9701/src/cpp/desktop/CMakeLists.txt#L681

Reply:
Ah I missed that. Makes sense to have it go through cmake before feeding it to the Electron packaging.

Reply:
It's running through CMake first now and I see that the package now has the `Info.plist` with the variables filled in.